learn/classes/object_instantiation.md

```diff
@@ -12,17 +12,75 @@ A class constructor in Python is a special method that is executed when an objec
 To instantiate a class in Python, we create an object of the class, which will call the constructor method. Here's an example of a simple class and how to instantiate an object of that class.
 
 ```python
-class Person:
-    def __init__(self, name, age):
+class Recipe:
+    def __init__(self, name, ingredients):
         self.name = name
-        self.age = age
+        self.ingredients = ingredients
+
+# Instantiate a Recipe object
+my_recipe = Recipe("Spaghetti Bolognese", ["spaghetti", "tomato sauce", "ground beef"])
+
+# Access the object's attributes
+print("Recipe Name:", my_recipe.name)
+print("Ingredients:", my_recipe.ingredients)
+```
+
+In the above example, the `Recipe` class has a constructor that sets the attributes `name` and `ingredients` for each new object that is instantiated. The `my_recipe` object is instantiated with the name "Spaghetti Bolognese" and a list of ingredients. The print statements will output "Recipe Name: Spaghetti Bolognese" and "Ingredients: ['spaghetti', 'tomato sauce', 'ground beef']."
```

Comment:
Highlight this output as a code : "Recipe Name: Spaghetti Bolognese" and "Ingredients: ['spaghetti', 'tomato sauce', 'ground beef']."